app/views/users/roles/edit.html.erb

```diff
@@ -46,22 +46,11 @@
           text: "This user's role is set to #{@user.role.humanize}. They are currently exempted from 2-step verification, meaning that their role cannot be changed as admins are required to have 2-step verification.",
         } %>
       <% else %>
-        <% role_hint = capture do %>
-          <%= render "govuk_publishing_components/components/list", {
-            visible_counters: true,
-            items: [
-              "<strong>Superadmins</strong> can create and edit all user types and edit applications.",
-              "<strong>Admins</strong> can create and edit normal users.",
-              "<strong>Super Organisation Admins</strong> can unlock and unsuspend their organisation and related organisation accounts.",
-              "<strong>Organisation Admins</strong> can unlock and unsuspend their organisation accounts.",
-            ]
-          } %>
-        <% end %>
```

Comment:
Not one for this PR but I wonder whether this highlights that we've missed this from the account edit role page?

Reply:
Yeah, maybe - although I think that page was more explicitly "designed", so it might have been intentional. In this case I've just tried to preserve the existing functionality/copy as closely as possible while still using the GOV.UK Design System.

Reply:
Addressed in https://github.com/alphagov/signon/pull/2537/commits/a0e540a4a064eaff66e206892822649c278020a1